qemu/tests/block_resize.py

```diff
@@ -1,14 +1,16 @@
 import logging
 import re
 
-from autotest.client.shared import error
```

Comment:
Obviously,
We have different opinions about it.
The tp-qemu is still kept in autotest organization.【autotest/tp-qemu】,
and it hasn't been *definitely* declared to shift to avocado now.
Maybe we need a copy of tp-qemu/tp-avocado for avocado to thoroughly solve the problem.